UM-USElab/gradecraft-development
# 2950 Set passwords manually - first pass

### Status
**IN DEVELOPMENT**

### Description
As a K-12 instructor, I need a way to manually set/reset someone else's password.

### Related PRs
N/A


### Todos


### Deploy Notes
N/A

### Gem dependencies
N/A

### Migrations
NO

### Steps to Test or Reproduce
As an administrator, navigate to learners, unique student and click "Edit" in the user's profile, then fill out change password fields and click the "Update User" button. Upon resetting a user's password, current user should see alert reading "[User]'s profile was successfully updated!".


### Impacted Areas in Application

* Administrator Views of Students

======================
Closes #[2950]
